src/anemoi/graphs/generate/tri_icosahedron.py

```diff
@@ -270,3 +281,24 @@ def add_neigbours_edges(
             graph.add_edge(node_neighbour, node)
 
     return graph
+
+
+def create_node_neighbors_list(
```

Comment:
I would probably use neighbours for consistency with the rest of the package and the config file.